stack_data/core.py

```diff
@@ -678,7 +676,10 @@ def _executing_node_common_indent(self) -> int:
                 indent = max(rang.start, begin_text)
                 indents.append(indent)
 
-        return min(indents) if indents else 0
+        if len(indents) <= 1:
+            return 0
+
+        return min(indents[1:])
```

Comment:
I don't really understand what the goal of this new approach is, but this line in particular looks like it's assuming the the current line is the first highlighted line. Remember that isn't always the case https://github.com/alexmojaki/stack_data/pull/20#discussion_r803015541

Reply:
No, this line is assuming that the first highlighted line might be highlighted with a different leading indentation than all the lines below it. 
With Python 3.9:
```
Traceback (most recent call last):
 File "C:\Users\Andre\github\stack_data\example.py", line 5, in <module>
       1 | import stack_data
       3 | stack_data.Formatter().set_hook()
-->    5 | nb = len(letter
                ^^^^^^^^^^
       6 |          for letter
                    ^^^^^^^^^^
       7 |          in "word")
                    ^^^^^^^^^^
TypeError: object of type 'generator' has no len()
```
With Python 3.6:
```
Traceback (most recent call last):
 File "C:\Users\Andre\github\stack_data\example.py", line 7, in <module>
       1 | import stack_data
       3 | stack_data.Formatter().set_hook()
       5 | nb = len(letter
                ^^^^^^^^^^
       6 |          for letter
                    ^^^^^^^^^^
-->    7 |          in "word")
                    ^^^^^^^^^^
TypeError: object of type 'generator' has no len()
```